### PR TITLE
Burrow idea attempt

### PR DESCRIPTION
**Pull request by Matuiss2**
Dont accept yet, ill put here for you to test

### CODE REVIEW

File: actions/army_control.py [hunk 1]
```diff
@@ -184,3 +189,14 @@ def attack_startlocation(self, unit):
         """It tell to attack the starting location"""
         if self.ai.enemy_start_locations:
             self.ai.add_action(unit.attack(self.ai.enemy_start_locations[0]))
+
+    async def burrow_zerglings_group(self):
```
Comment (Helfull) [inline actions/army_control.py]: could we make this another action?

Comment (Helfull) [reply actions/army_control.py]: e.g.: `BlockEnemyExpansions`

Comment (tweakimp) [reply actions/army_control.py]: will do after i make it work 100%

Comment (tweakimp) [reply actions/army_control.py]: done

**Pull request merged**